src/openrct2-ui/windows/TitleScenarioSelect.cpp

```diff
@@ -462,7 +459,10 @@ static void window_scenarioselect_paint(rct_window* w, rct_drawpixelinfo* dpi)
         { // old-style
             ft.Add<rct_string_id>(ScenarioCategoryStringIds[i]);
         }
-        gfx_draw_string_centred_wrapped(dpi, gCommonFormatArgs, x, y, 87, format, COLOUR_AQUAMARINE);
+
+        ScreenCoordsXY stringCoords(
+            (widget->left + widget->right) / 2 + w->windowPos.x, (widget->top + widget->bottom) / 2 + w->windowPos.y - 3);
```

Comment:
If you put a return after the comma, does clang-format accept that?

Reply:
No it doesn't, it reformats this way.